packages/styles/scss/patterns/blocks/content-block-media/README.md

```diff
@@ -5,41 +5,12 @@
 Import the package css into the top of your main CSS file.
```

Comment:
Out of curiosity what's going on in this file?

Reply:
I don't think it even needs to be there. I'll remove